docs: example of how to change releasable units (#1335)
googleapis/release-please: docs/customizing.md

To add a new changelog type, create a new class that implements the
[`ChangelogNotes` interface](/src/changelog-notes.ts).

> ### Customizing Changelog Types and Releaseable Units

Reviewer: ```suggestion
### Customizing Changelog Sections/Headings
```

> To change the "releasable units" for your code, you can [customise the changelog sections](https://github.com/google-github-actions/release-please-action#overriding-the-changelog-sections) for your project.

Reviewer: Perhaps something along the lines of: "A release pull request is created when a commits lands that would create an entry in the changelog. You can customize which types of commits appear in the changelog by updating the `changelog-sections` configuration."

> For example, if I wanted to release when I have `docs` changes on my Node.js project, I could update the GitHub Actions Workflow like so:

Reviewer: ```suggestion
For example, to include `docs` changes, configure the `changelog-types` configuration in the GitHub Actions Workflow like so:
```

We'd like to avoid using first-person in the guides.
